BuyPolish/Pola/UI/ProductSearch/ReportProblem/ReportProblemViewController.swift

```diff
@@ -9,6 +9,7 @@ final class ReportProblemViewController: UIViewController {
     private let reason: ReportProblemReason
     private let analytics: AnalyticsHelper
     private var imageCount: Int = 0
+    private var sendImageEnabled: Bool = false
```

Comment:
`let` zamiast `var`. Co ty na to by ustawiać to w inicie z domyślnym `false`? Można by wtedy zrobić testy unitowe z wykorzystaniem snapshotów dla obu wersji.